gemspec.yml

```diff
@@ -1,7 +1,7 @@
 name: bundler-audit
 summary: Patch-level verification for Bundler
 description: bundler-audit provides patch-level verification for Bundled apps.
-license: GPL-3.0-or-later
+licenses: GPL-3.0-or-later
```

Comment:
If the gem only has one license, why use `licenses`?

Reply:
But then, why you use `authors`, when here is also only one? I think consistency to gem specification make sens here. That's why `licenses` and `authors`.

Reply:
OK, I reverted `licenses` change.